src/edu/wright/cs/raiderplanner/controller/MainController.java

```diff
@@ -100,7 +102,22 @@ public static void setSpc(StudyPlannerController newSpc) {
 	 */
 	public static void initialise() {
 		try {
-			ui.showStartup();
+			if (settings.getAccountStartup() == true) {
+				try {
+					File file = new File(settings.getDefaultFilePath());
+					if (file.exists() && !file.isDirectory()) {
+						plannerFile = file;
+					} else {
+						ui.showStartup();
+					}
+				} catch (Exception e) {
```

Comment:
You should not catch `Exception` in this manner. Catch the specific type, most likely `FileNotFoundException` or possibly `IOException` instead.

Reply:
I have fixed this and committed the change. I will push it to this pull request once I know what to do regarding the gitignore.